apis/metal3.io/v1alpha1/doc.go

```diff
@@ -0,0 +1,19 @@
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package v1alpha1 contains API Schema definitions for the metal3.io v1alpha1 API group
+// +kubebuilder:object:generate=true
```

Comment:
Yes, definitely. In my case I don't need those because I'm using a customized filter function for kube-openapi.
But for the standard usecase it's most definitely required.

Reply:
Please add
```
// +k8s:openapi-gen=true
```

Reply:
It was already added

Reply:
Oh, sorry! I missed it since it was below the comment :facepalm: